source/patterns/04-pages/research-article/research-article~en.yaml

```diff
@@ -32,6 +32,48 @@ contentHeader:
                 attributes:
                   datetime: '2018-12-18'
                 text: Dec 18, 2018
+  authors:
+    attributes:
+      aria-label: Authors of this article
+    items:
+      - content:
+          text: Andrea Campbell
```

Comment:
I've kept/translated the original affiliations, but replaced real names with random ones produced by https://uinames.com/.